### PR TITLE
Units, docs(web): add cases for testing direction flags of langdef option

### PR DESCRIPTION
**Pull request by masatake**
Surprisingly I didn't write the test cases for them.
The new test cases are based on https://docs.ctags.io/en/latest/running-multi-parsers.html#direction-flags.

More surprisingly the code works as written in the document:-)

### CODE REVIEW

**Comment by masatake** [comment]
@hirooih, I wrote test cases based on the document you reviewed.
The cases are simple enough so they will help readers understand the documents.

**Review by hirooih — CHANGES_REQUESTED** [review]
Units tests are OK.

At least add a "more".

File: docs/running-multi-parsers.rst [hunk 1]
```diff
@@ -310,6 +318,9 @@ The following three direction flags are available;
 	``input.sub``   recorded              recorded
 	=============== ===================== ======================
 
+	An example is at "`Units/flags-langdef-directions.r/bidirectional
+	<https://github.com/universal-ctags/ctags/tree/master/Units/flags-langdef-directions.r/bidirectional>'".
+
 If no direction flag is specified, it implies ``{shared}``.
 
 See ":ref:`optlib_directions`" in ":ref:`optlib`" for examples of using the
```
[the remark this answers is not in the document]
Comment (masatake) [reply docs/running-multi-parsers.rst:326]: I agree with you. I will update the document again.

[the remark this answers is not in the document]
Comment (masatake) [reply docs/running-multi-parsers.rst:326]: I didn't remember the examples were in optlib.rst. Thank you for pointing out this.
So I withdraw the change for docs/running-multi-parsers.rst.

I will just introduce "TESTCASE:" flag.

**Review by hirooih — APPROVED** [review]
I see.  This is another good selection.

**Pull request merged**